README.md

```diff
@@ -50,13 +51,11 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
 
-<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+[Live Demo Link](https://budget-app-9ny9.onrender.com/)
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure that:
1. The project is deployed and accessible online, (you can use Heroku, Netlify or GitHub Pages).
2. The project is deployed properly, and the home and detail page works as expected.
3. A link to the deployed project is present in the README.
- Kindly address this 💯 👍🏻

Reply:
@BenMukebo, the link to the deployed project is in my readme file